README.md

```diff
@@ -367,3 +445,122 @@ You will receive a JSON object in response. Using the `_id` property, you can cr
 ```
 
 Set `yourImageId` to the `_id` of the object you received when you POSTed to `/api/v1/apostrophe-images`.
+
+## Working with pages
+
+The examples above all concern pieces. Pieces are the most natural candidate for a REST API, but you can also use `apostrophe-headless` to work with pages:
+
+```javascript
+modules: {
+
+  'apostrophe-headless': {},
+
+  'apostrophe-pages': {
+    restApi: true
+  }
+}
+```
+
+## Retrieving the home page and its children
+
+Now your app can access:
+
+`/api/v1/pages`
+
+To get information about the home page and its children. The response isa single JSON object with `slug`, `path`, `title`, `type`, `_url` and other properties describing the home page, similar to the way pieces are returned (see the "products" examples above). In addition, information about children of the home page is returned.
+
+### Accessing child pages
+
+Basic information about the top-level children of the home page (aka the "tabs" of your site) is available in the `_children` property of the returned object. This property is an array. Each element has, at a minimum, `_id`, `title`, `type` and `slug` properties.
+
+### Fetching detailed information about one page
+
+Armed with the `_id`, you can obtain detailed information about a page by making a separate API request:
+
+`/api/v1/pages/ID_GOES_HERE`
+
+A page returned in this way may in turn offer its own `_children` property.
+
+This response will include schema fields, areas, etc. in the same detail as it would when requesting a piece.
+
+*The `_children` property does not necessarily exist if there are no child pages.*
```

Comment:
This is lame, make sure _children exists, even if empty, less work for consumer.